Add mobile menu

s2jack opened the pull request:
Add mobile menu elements
Add close function
Add open menu function
Add ul and li items to navigate  related sections
Add main.js

s2jack committed:
Set workplace and menu items for mobile
Add linters our local environment
Add main js and declare into html
Add mobile menu and some css

Commit:
Add main.js and style for ul and li

s2jack committed:
Fix some linter errors

Fix linter errors

TuyishimireEric commented on styles.css:
```diff
@@ -65,6 +65,65 @@ main {
   margin-right: 16px;
 }
 
+#mobile_menu {
+  height: 568px;
+  width: 100vw;
+  display: none;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+#mobile_menu_items {
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+}
+
```
- [ ] **Good job on implementing the mobile menu 👍 👍** 
  however, it would be better to use a fixed position for the mobile menu items as using a ```fixed position``` for the mobile menu can improve the user experience, as it ensures that the menu remains accessible while scrolling through the page content. 😎😎
 
   ![image](https://user-images.githubusercontent.com/102757126/217014312-7d48c197-c575-4b1f-9e7c-3845bd4013ce.png)

   _It can also help provide a more consistent and predictable navigation experience. You may follow this[ link](https://developer.mozilla.org/en-US/docs/Web/CSS/position) for more info 👍_

TuyishimireEric commented on main.js:
```diff
@@ -0,0 +1,25 @@
+const hamimage = document.querySelector("#menubutton");
+const shape01 = document.querySelector("#shape01");
+const mobileMenu = document.querySelector("#mobile_menu");
+const homeSec = document.querySelector("#home_sec");
+const crossImg = document.querySelector('#close_icon');
+const mobMenuItems = document.querySelectorAll(".mob_menu_item");
+
+const menuOpen = () => {
+  hamimage.style.display = 'none';
+  shape01.style.display = "none";
+  mobileMenu.style.display = "flex";
+  homeSec.style.visibility = "hidden";
+}
+
+const menuClose = ()=> {
+   hamimage.style.display = 'flex';
+  shape01.style.display = "flex";
+  mobileMenu.style.display = "none";
+  homeSec.style.visibility = "visible";
+}
+
+crossImg.addEventListener('click',menuClose);
+mobMenuItems.forEach(item => {
+  item.addEventListener('click',menuClose);
+});
```
- [ ] in this project, it would be better to set up both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository and also remember to update your ```linter.yml``` file so that your PR checks will also include ```eslint```. 😎😎
   ## Currently 
   ![image](https://user-images.githubusercontent.com/102757126/216946692-69381b35-5b6c-4da7-8bca-4af7ab595c4b.png)

   _Remember, setting up linters in a project repository is a good practice to ensure the code is written following a consistent style and adheres to best practices. JavaScript and HTML&CSS linters can help identify syntax errors, typo errors, and other issues that might negatively impact the quality and functionality of the code. 👍👍_

TuyishimireEric reviewed (changes requested):
# Status: REQUIRED CHANGES :recycle: 
### Hi  @Team 👋 👋 
#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
There are some issues that you still need to work on to go to the next project but you are almost there!
![](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)

## Highlights 
✔️ Professional PR. 👍
✔️ No linter errors👍
✔️ Professional README👍

## Required Changes ♻️

- [ ] **Check the comments below 👇**

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

s2jack committed:
Add some styles - eslint
Add eslint to linters
Fix position of mobile menu
Fix some style errors on mobile version

s2jack committed:
Fix eslint errors

Olamarx reviewed (approved):
Hi @s2jack, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
<img src="https://media2.giphy.com/media/jJQC2puVZpTMO4vUs0/giphy.gif"/>
## Highlights
- Linters passed ✔️ 
- Mobile menu functionalities ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
